src/core/data-tests.js

```diff
@@ -98,12 +98,22 @@ export function run(conf) {
           }
           return href;
         });
+      const duplicates = testURLs.filter(
+        (links, i, self) => self.indexOf(links) !== i
+      );
+      if (duplicates) {
+        pub(
```

Comment:
We really recently added a new `showInlineWarning` util function that might be good to use here instead. It's shows "inline warnings" on the element itself, which is kinda nice. 

For the message, maybe:
 
```JS
showInlineWarning(
  elem, 
  `Duplicate tests found.`
  `To fix, remove duplicates from "data-tests": ${duplicates.join(", ")}`
);
```

That should add a red squiggly underneath the offending element. See if the message makes sense when you hover over the element.